clap-utils/src/input_parsers.rs

```diff
@@ -99,9 +99,10 @@ pub fn pubkeys_sigs_of(matches: &ArgMatches<'_>, name: &str) -> Option<Vec<(Pubk
 pub fn signer_of(
     name: &str,
     matches: &ArgMatches<'_>,
+    wallet_manager: &Option<Arc<RemoteWalletManager>>,
```

Comment:
These could be `Option<&Arc<RemoteWalletManager>>` to spare us the eventual awkward `&None`.  Caller just needs to `.as_deref()`.